<?xml version="1.0"?>
<launch>
    <node pkg="object_detection" exec="front_cam_detector" name="front_cam_detector">
        <param name="class_names" value="['gate', 'octagon_table', 'octagon_top', 'path_marker', 'sawfish', 'shark']"/>
        <param name="frontcam_model_path" value="/ros2_ws/src/vision/model_pipeline/models/frontcam.pt"/>
        <param name="frontcam_input_topic" value="/vision/front_cam/image_enhanced"/>
        <param name="frontcam_output_topic" value="/vision/front_cam/object_detection"/>
        <param name="queue_size" value="10"/>
    </node>
    <node pkg="object_detection" exec="down_cam_detector" name="down_cam_detector">
        <param name="class_names" value="['gate', 'octagon_table', 'octagon_top', 'path_marker', 'sawfish', 'shark']"/>
        <param name="downcam_model_path" value="/ros2_ws/src/vision/model_pipeline/models/downcam.pt"/>
        <param name="downcam_input_topic" value="/vision/down_cam/image_enhanced"/>
        <param name="downcam_output_topic" value="/vision/down_cam/object_detection"/>
        <param name="queue_size" value="10"/>
    </node>
</launch>